apps/developer-portal/src/components/users/user-profile.tsx

```diff
@@ -269,7 +269,7 @@ export const UserProfile: FunctionComponent<UserProfilePropsInterface> = (
                             </Form.Field>
                         ) : (
                             <Field
-                                data-testid={ `${ testId }-profile-form-${ schema.name }-input` }
+                                data-testid={ `${ testId }_profile_form_${ schema.name }_input` }
```

Comment:
Shall we go with dashes instead of underscores?

Reply:
Fixed with f762024